app/models/project_group.rb

```diff
@@ -0,0 +1,14 @@
+class ProjectGroup < ActiveRecord::Base
+  validates_presence_of :title, :team_id
+
```

Comment:
@caiosba I think we should also add a validation for uniqueness title per team

Reply:
Thanks Sawy, done!